schemas/policy.json

```diff
@@ -0,0 +1,26 @@
+{ "$id": "https://izens.net/schemas/policy.json",
+
+  "definitions": {
+      "action": {
+          "type": "object",
+          "properties": {
+              "action": { "type": "string", "enum": ["block", "warn"] },
```

Comment:
Do we intend to have a different action type for "boycott"? I think that's a discussion we should have. We either always send the boycott request when the active is block and/or warn, or we have a separate action for it. We can obviously add the later, if we're not sure.

Reply:
Another unrelated comment that can also be changed later: maybe it should be { actions: [{ type: "warn", message: "bad website" }]. As in, maybe this property should be  `type` instead of `action`. Just doesn't seem right for an action to have an action, once defined as a schema. But currently not how we have implemented, of course.